src/translations/dist/en.json

```diff
@@ -842,21 +1062,31 @@
     "paragliding_takeoff": {
       "waypoint_types": "paragliding takeoff"
     },
+    "Parameters of the Digital Camera": "Détails on the camera",
     "parking_fee": "parking_fee",
     "partial_trip": "Partial trip",
     "participant_count": "Number of participants",
     "participants": "Participants",
+    "Participants and context": "Participants and context",
     "partly_protected": {
       "rain_proof_types": "partly protected"
     },
     "pass": {
       "waypoint_types": "pass"
     },
     "Password": "Password",
-    "pedestrian access": "pedestrian access",
```

Comment:
should not be removed?

Reply:
It has been replaced by a version with a context : https://github.com/c2corg/c2c_ui/pull/594/files/b685fb16f143b589e53909a5372d970e9a1e9ead#diff-14307f8f25eb7db0add8c8f57463ffb7R1081